tests/test_fileio/test_base.py

```diff
@@ -0,0 +1,103 @@
+"""
+Tests for fileio.base module
+"""
+
+# author: Andrew R. McCluskey (arm61)
+
+from orsopy.fileio.base import ValueVector
+import unittest
+from numpy.testing import assert_almost_equal, assert_equal
```

Comment:
For non-array objects I would just use the assert attributes of the TestCase class. We should definitely  check that floats/ints are actually floats and not numpy scalar objects, as these lead to issues with yaml when writing.